generators/gae/index.js

```diff
@@ -0,0 +1,694 @@
+                        done();
+                    });
+                });
+            },
+
+            promptForCloudSqlDatabaseNameIfNeeded() {
+                if (this.abort) return;
+                if (this.gcpCloudSqlInstanceName !== 'new' && this.gcpCloudSqlDatabaseName) return;
+
+                const done = this.async();
+
+                const prompts = [
+                    {
+                        type: 'input',
+                        name: 'gcpCloudSqlDatabaseName',
+                        message: 'Google Cloud SQL Database Name',
+                        default: this.gcpCloudSqlDatabaseName ? this.gcpCloudSqlDatabaseName : this.baseName
+                    }];
+
+                this.prompt(prompts).then((props) => {
+                    this.gcpCloudSqlDatabaseName = props.gcpCloudSqlDatabaseName;
+                    this.gcpCloudSqlDatabaseNameExists = false;
+                    done();
+                });
+            }
+
+        };
+    }
+
+    get default() {
+        return {
+        };
+    }
+
+    get configuring() {
+        return {
+            insight() {
+                const insight = this.insight();
+                insight.trackWithEvent('generator', 'gae');
+            },
+
+            configureProject() {
+                if (this.abort) return;
+                const done = this.async();
+
+                if (!this.gaeLocationExists) {
+                    this.log(chalk.bold(`Configuring Google App Engine Location "${chalk.cyanthis.gaeLocation}"`));
+                    exec(`gcloud app create --region="${this.gaeLocation}" --project="${this.gaeProjectId}"`, (err, stdout) => {
+                        if (err) {
+                            this.log.error(err);
+                            this.abort = true;
+                        }
+
+                        done();
+                    });
+                } else {
+                    done();
+                }
+            },
+
+            createCloudSqlInstance() {
+                if (this.abort) return;
+                if (!this.gcpCloudSqlInstanceName) return;
+                if (this.gcpCloudSqlInstanceNameExists) return;
+                const done = this.async();
+
+                this.log(chalk.bold('\nCreating New Cloud SQL Instance'));
+
+                const name = this.gcpCloudSqlInstanceName;
+
+                const cmd = `gcloud sql instances create "${name}" --region='${this.gaeLocation}' --project=${this.gcpProjectId}`;
+                this.log(chalk.bold(`\n... Running: ${cmd}`));
+
+                exec(cmd, (err, stdout, stderr) => {
+                    if (err) {
+                        this.abort = true;
+                        this.log.error(err);
+                    }
+
+                    this.gcpCloudSqlInstanceName = execSync(`gcloud sql instances describe jhipster --format="value(connectionName)" --project="${this.gcpProjectId}"`, { encoding: 'utf8' });
```

Comment:
How can this works fine if the database instance is called something different from "jhipster" ?

Reply:
Yes it looks like a mistake. If you have improvements to propose, please do a PR 🙏 .

Reply:
@PierreBesson The fix as already been done and merged in master ;)

Reply:
yes I already did it ;)